.github/workflows/e3sm-gh-pages.yml

```diff
@@ -0,0 +1,50 @@
+name: Build and deploy gh-pages branch with Mkdocs
+
+on:
+  # Runs every time master branch is updated
+  push:
+    branches: ["master"]
+  # Runs every time a PR is open against master
+  pull_request:
+    branches: ["master"]
+  workflow_dispatch:
+
+concurrency:
+  # Prevent 2+ copies of this workflow from running concurrently
+  group: e3sm-docs-action
+  
+jobs:
+  Build-and-Deploy-docs:
```

Comment:
We'd still need to disable this action, even if we integrated eamxx docs upstream. The reason is that e3sm may (big conditional) update the docs, so we don't want to overwrite at every scream/master update, since we would likely erase what was added upstream.